src/cli/cli.ts

```diff
@@ -90,6 +90,8 @@ program
           process.exit(1);
         }
       }
+      if (browserType.length && browserType.includes('chromium'))
```

Comment:
Let's move this logic to installBrowsersWithProgressBar instead? This way we won't miss ffpmeg in some other usecase.